docs/dashboards/dashboards-overview.asciidoc

```diff
@@ -6,12 +6,18 @@
 :frontmatter-tags-content-type: [overview]
 :frontmatter-tags-user-goals: [visualize, monitor, analyze]
 
-The following sections describe the {security-app}'s prebuilt dashboards, which provide visualizations of your security environment.
+This section documents the {security-app}'s default dashboards, which provide visualizations of your security environment. To view them in {security-app}, select *Dashboards* from the navigation menu.
+From the Dashboards page, you can access the default dashboards, as well as create and access custom dashboards.
 
-You can also create and access custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard**. Custom dashboards must have the tag `Security Solution` to appear on the Dashboards page.
+To create a new custom dashboard, click **Create Dashboard**. You can control which custom dashboards appear in the table by:
+* Using the text search field to filter by name or description
+* Using the **Tags** menu to filter by tag
+* Clicking one of a custom dashboard's tags to toggle whether to filter for that tag
+To create a new tag or edit existing tags, open the **Tags** menu and click **Manage tags**.
 
 image::images/dashboards-landing-page.png[The Dashboards landing page]
 
+For more information about each of the default dashboards, refer to the other pages in the Dashboards section. For more information about creating custom dashboards, refer to {kibana-ref}/create-a-dashboard-of-panels-with-web-server-data.html[Create your first Kibana dashboard].
```

Comment:
Is the first sentence pointing users towards docs for the other dashboards? If yes, it might be more clear if you replace "pages" with "documentation". Here's an example:

```suggestion
Refer to documentation for the other {elastic-sec} dashboards to learn more about them. For more information about creating custom dashboards, refer to {kibana-ref}/create-a-dashboard-of-panels-with-web-server-data.html[Create your first Kibana dashboard].
```